src/explore-education-statistics-admin/src/pages/bau/BauImportsPage.tsx

```diff
@@ -29,12 +29,13 @@ const BauImportsPage = () => {
               Subject title
             </th> */}
               <th scope="col">Subject ID</th>
+              <th scope="col">Release title</th>
```

Comment:
I'd suggest merging the last column into this one, as it's a bit redundant to show the release title twice